### PR TITLE
Site admin language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Better, but not quite there yet.
Edit: What is a "voutput"?

### CODE REVIEW

File: src/web/site_admin.ml [hunk 1]
```diff
@@ -1090,8 +1090,8 @@ let () =
             in
             se.se_public_creds_received <- true;
             let service = preapply ~service:election_draft uuid in
-            Pages_common.generic_page ~title:(s_ "Success") ~service
-              (s_ "Credentials have been generated and mailed! You should download private credentials (and store them securely), in case someone loses his/her credential.") () >>= Html.send
+            Pages_common.generic_page ~title:(s_ "Done") ~service
+              (s_ "Credentials generated and sent via e-mail. Download your private credentials (and store them securely), in case someone loses theirs.") () >>= Html.send
           )
```
Comment (glondu) [inline src/web/site_admin.ml:1095]: This is a misinterpretation: here, the private credentials are all the voters', and each individual voter can lose their credential.

File: src/web/site_admin.ml [hunk 2]
```diff
@@ -2406,14 +2406,14 @@ let () =
...
+                    "Could not change the password of %S (%S) for the %s service"
                     username email service
                 )
            | Some (username, address) ->
               Web_signup.send_changepw_link ~service ~address ~username
          in
          let message =
-           s_ "If the account exists, an e-mail was sent with a confirmation link. Please click on it to change your password."
+           s_ "If the account exists, an e-mail containing a confirmation link will be sent. Please click it to change your password."
          in
```
Comment (glondu) [inline src/web/site_admin.ml:2417]: Why did you change the tense?

**Review by glondu — COMMENTED** [review]
Thank you for your suggestions. However, I am a bit worried that this destroys existing translations and manually merging your changes with existing translations looks appalling.

**Comment by glondu** [comment]
> What is a "voutput"?

It is an internal data structure used in the distributed key generation (trustees in threshold mode). It should not appear in the user interface. Does it?

File: src/web/site_admin.ml (hunk 2 above)
Comment (comradekingu) [reply src/web/site_admin.ml:2417]: Because of the former part, and also it should have been something like "if the account was found to exist, an e-mail would have been sent" or something, but that is complex.

**Comment by comradekingu** [comment]
@glondu I falsely remembered seeing it in the strings.

**Comment by glondu** [comment]
First of all, thank you for all your contributions to Belenios!

> I am a bit worried that this destroys existing translations and manually merging your changes with existing translations looks appalling.

I found a solution, suggested in Weblate documentation: I have set source language to "English (developper)" in Weblate, and added "English" as a translation, to avoid breaking all translations when the English one is changed. The idea is that the source strings are set in stone.

Please make your changes in Weblate.
